packages/blockchain-ui/src/theme/colors.ts

```diff
@@ -65,4 +77,26 @@ export const colors = {
   error900: "#782424",
 };
 
-
+/**
+ * All the theme colors declarations here..
+ */
+export const themeColors: Record<TColorSchemes, TSchemeColors> = {
+  dark: {
+    text: {
+      primary: "#FFFFFF",
```

Comment:
These Colors are already present in Color Pallette If your search the color in code you will find name.
Also component specific colors are stored theme.component.BuiComponentName.common.color , etc